src/OrchardCore/OrchardCore.Mvc.Core/SharedViewCompilerProvider.cs

```diff
@@ -20,6 +21,8 @@ namespace OrchardCore.Mvc
 {
     public class SharedViewCompilerProvider : IViewCompilerProvider
     {
+        private readonly static IMemoryCache _cache = new MemoryCache(new MemoryCacheOptions());
```

Comment:
`static`? I imagine the type is a singleton and so you wouldn't need this to be `static`?

Reply:
We want to share the instance among tenants, which have their own DI container, so `static`